peps/pep-0729.rst

```diff
@@ -0,0 +1,414 @@
+* The concept of `recursive aliases <https://github.com/python/typing/issues/182>`_
+* Semantics of `variable initialization <https://mail.python.org/archives/list/typing-sig@python.org/thread/GYVM5KEE6URE6PAH7UTK6324M7GWSFQS/#SY3KPJCAW4UTOOCH3XRJYROSGDEGOTWI>`_
+* `Reachability semantics <https://github.com/python/mypy/issues/15158#issuecomment-1677915108>`__
+  of annotations on ``__exit__``
+* `Symbol visibility <https://mail.python.org/archives/list/typing-sig@python.org/thread/YLJPWECBNPD2K4TRIBRIPISNUZJCRREY/#OX4GLBQOOCMRE5YPZEY3R3XNV6DD7XLW>`_
+* Use of `NoReturn for exhaustiveness checking <https://github.com/python/mypy/issues/5818>`_
+
+The Steering Council is not well-placed to solve the above problems
+-------------------------------------------------------------------
+
+The SC has the entire language in its remit, and is not well-placed to make
+decisions that are purely about the type system -- if only because they don't have
+the time to deal with type system arcana alongside their other responsibilities.
+This is similar in spirit to the reasons why the Steering Council sometimes uses
+PEP delegation.
+
+Specification
+=============
+
+We propose the creation of a new group, the Typing Council. This group will
+be responsible for developing and maintaining the Python type system, and
+for solving the above problems.
+
+The "operations and process" section describes how this group would operate and
+be governed.
+
+The more exciting "projects" section describes solutions to the above problems
+that the Typing Council could shepherd.
+
+Mandate
+-------
+
+The Typing Council's mandate is to ensure that the Python type system is:
+
+* **Useful**: The type system should serve common use cases. As identified
+  by :pep:`484`, the primary use case is static analysis, but there are others,
+  such as runtime type checking, static compilation, IDE support, and documentation.
+  The Typing Council should consider all of these use cases when making decisions,
+  and be open to supporting additional use cases as they come up.
+* **Usable**: The type system should be easy to use for Python developers. It
+  should be ergonomic to write well-typed Python code that is accepted by type
+  checkers. There should be good documentation for the type system.
+* **Stable**: As the type system matures, users should be able to rely on their
+  typed code continuing to work and be able to trust their mental model for the
+  type system. Changes should be made with care and in a way
+  that minimizes disruption. Nevertheless, the type system should be able to
+  evolve, and type checkers need not be held to the same strict compatibility
+  standards as the Python language itself.
+
+Operations and process
+----------------------
+
+The council would have three members, comprised of prominent community members,
```

Comment:
Yeah, but I don't think that the EB is slow because we're trying to get too many people involved. We're just all five of us a bit busy. :-)

Maybe you could think who you'd ask to be on the council and discuss your choices with the SC (and with your selected candidates)?

Reply:
Good point, I will do that